backends/tfhe-cuda-backend/cuda/src/integer/compression/compression.cuh

```diff
@@ -246,6 +255,9 @@ host_integer_decompress(cudaStream_t const *streams,
                         uint32_t const *h_indexes_array, void *const *d_bsks,
                         int_decompression<Torus> *h_mem_ptr) {
 
+  static_assert(std::is_same_v<Torus, uint64_t> ||
```

Comment:
Now on main I think we should use GPU_ASSERT for this cc @andrei-stoian-zama

Reply:
"static_assert performs this check at compile time. GPU_ASSERT does not. I don't think it makes any sense to check this during execution, since we don't want that function to be compiled for __uint128_t."

Reply:
Ah totally thanks!